google/cloud/firestore_v1/order.py

```diff
@@ -51,6 +52,13 @@ def from_value(value) -> Any:
 
         if v not in lut:
             raise ValueError(f"Could not detect value type for {v}")
+
```

Comment:
nit: this isn't from you, but I feel like the `lut` variable should be moved outside the function into a constant. No need to re-create it each time we call from_value

Then, instead of checking `if v not in lut`, we could just catch the KeyError to raise the ValueError above

Reply:
I do not have previous context though I thought we need to keep lut in scope since lut is referring the value in TypeOrder. Otherwise it's a circular dependency: TypeOrder -> LUT -> TypeOrder.